src/main/java/eu/mikroskeem/worldeditcui/render/IrisPipelineProvider.java

```diff
@@ -1,68 +0,0 @@
-package eu.mikroskeem.worldeditcui.render;
-
-import net.coderbot.iris.Iris;
-import net.coderbot.iris.layer.GbufferProgram;
-import net.coderbot.iris.layer.GbufferPrograms;
-import net.coderbot.iris.pipeline.WorldRenderingPipeline;
-import net.coderbot.iris.pipeline.newshader.CoreWorldRenderingPipeline;
-import net.fabricmc.loader.api.FabricLoader;
-import net.minecraft.client.render.GameRenderer;
-import net.minecraft.client.render.Shader;
-import net.minecraft.client.render.VertexFormat;
-import net.minecraft.client.render.VertexFormats;
-
-public final class IrisPipelineProvider implements PipelineProvider {
```

Comment:
why was this deleted? is it no longer necessary?

Reply:
I spoke with Iris and we confirmed it wasn't needed anymore.